cleanlab/classification.py

```diff
@@ -404,105 +556,21 @@ class label.
             self.confident_joint[self.pulearning][1 - self.pulearning] = 0
             self.confident_joint[1 - self.pulearning][1 - self.pulearning] = 1
 
-        # This is the actual work of this function.
         if self.verbose:
             print("Using predicted probabilities to identify label issues ...")
-        # Get the indices of the examples we wish to prune
+        # Get the indices of label issues
         self.label_issues_mask = find_label_issues(
```

Comment:
Change the import at the top to be `from cleanlab import filter` and then change this call everywhere to be `filter.find_label_issues()` to distinguish between the two.

Reply:
^^ do this everywhere. Otherwise its very hard to distinguish between the two functions that are named identically. You could also consider changing the function name so that its not identical.

Reply:
done